### DISCUSSION TITLE
Development Status as of Now

### DISCUSSION THREAD

**Discussion by seaglade** [KiARC/Sobriety#111] [Announcements]
I know I haven't updated Sobriety in a long time. I have not stopped caring about it, to be clear. There are a few compounding things that are making it difficult, namely:

- My old laptop died a while back, and although the disks aren't damaged, I haven't located the signing keys yet
- I'm still in school, and especially with college planning recently it's been difficult to keep up with my non-school projects
- I've had a *lot* to do in school too, particularly in my non-CS classes, and that's been pretty exhausting
- I also have other commitments to keep up with, including acting in the school musical, which take up a lot of time

That being said, Sobriety is not "no longer maintained". Consider it a hiatus for now. I'm hoping to resume active development once I've graduated and my days are less packed. I also would like to start other projects if I'm able to. I've always had this pipedream goal of building a whole suite of apps like Sobriety but for other parts of health (especially mental health), and I'm starting to feel more and more like I'll actually be able to do it someday. One big thing has been learning new programming frameworks in and between my classes. In the near future I'm considering trying to rewrite Sobriety to be more maintainable, manageable and extensible, but that will take time and be a huge change. That being said, I think I'll be able to do it. Thank you all for supporting Sobriety for all of this time. I hope that support can continue into the future.

**Comment by saltecaramel** [comment]
Take your time. We're looking forward to have you back:)

**Comment by yigicool** [comment]
Sending you good vibe for the school and any of your projects !

**Comment by yigicool** [comment]
Hi how is it going ?
I just came by to check if there's anything new, and it happens this announcement was posted exactly two years ago :joy: 
I hope your doing good and things are going well :smile:

**Reply by seaglade** [reply]
Things have been alright! Crazy how it's been 2 years already. Time flies, I guess. Unfortunately, Sobriety is definitely not getting any further updates - I never did recover the signing key, and the code isn't very maintainable (it was my first android project, you can imagine how that goes 🤣), so I've accepted that I'll just have to build a new app if and when I have enough time. I've still got that pipedream of a proper health suite, and at this point I have more of an idea of how it might come together (tauri+astro, probably), but as of right now it's still in the "todo-list in my notes app" stage.
